drivers/can/Kconfig

```diff
@@ -51,6 +51,17 @@ config CAN_NPENDINGRTR
 	---help---
 		The size of the list of pending RTR requests. Default: 4
 
+config CAN_TXCONFIRM
```

Comment:
Do you have an example implementation of this feature for any NuttX upstream architecture?

Reply:
Yes, 
Reason:
the upper app need to know whether this frame is already sent to bus after it send a frame to bus. so I develop this feature to notify the upper app this frame is sent to bus.
please check out the picture below.